apple-catalog-parsing/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "apple-catalog-parsing"
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+thiserror = "1.0.38"
```

We can use the latest version here


```suggestion
thiserror = "2.0.12"
```